src/pokemon.c

```diff
@@ -6669,6 +6669,12 @@ u16 GetEvolutionTargetSpecies(struct Pokemon *mon, u8 mode, u16 evolutionItem, s
                     targetSpecies = gEvolutionTable[species][i].targetSpecies;
                 }
                 break;
+            case EVO_LEVEL_FOG:
```

Comment:
Small nitpick, but I would the fog case next to the rain one, since they're so similar.

Reply:
Good idea, done.